pyproject.toml

```diff
@@ -353,10 +353,14 @@ reportMissingTypeArgument = false
 
 
 [tool.pytest.ini_options]
+minversion = "8.2"
+pythonpath = "." # legacy configuration, new pytest versions don't add the rootdir to path
 testpaths = "tests"
 addopts = "--strict-markers -Werror -s"
 xfail_strict = true
 asyncio_mode = "strict"
+asyncio_default_fixture_loop_scope = "function"
+filterwarnings= "default"
```

Comment:
seems like these two aren't needed? Removing them appears to make no difference when running tests, at least for me

Reply:
Planning well ahead. 

https://pytest-asyncio.readthedocs.io/en/latest/reference/configuration.html#asyncio-default-fixture-loop-scope